### PR TITLE
[update][fix][ui]: made minor bug fixes in streambuilder,scroll container and ui overflows

### PR DESCRIPTION
**Pull request by dhruv1294**
- Made marketStatus realtime in exchange page web.
- Fix scrollContainer attached to more than one scroll view error in exchangePage web
- Fix streamBuilder isNegative errors in home and company page
- Added bouncing scroll physics to exchange page
- Fixed UI overflow in trading bottom sheet
- added todos for trading bottom sheet which are important to fix

### CODE REVIEW

File: lib/pages/stock_exchange/components/stock_detail.dart [hunk 1]
```diff
@@ -123,6 +124,14 @@ class _StockDetailState extends State<StockDetail> {
   }
 
   Widget _marketStatus(Stock company) {
+    final Stream<Int64> dayHighStream =
```
Comment (Sudhindra3000) [inline lib/pages/stock_exchange/components/stock_detail.dart:127]:
```suggestion
    final dayHighStream =
```
No need to specify type ig

Comment (Sudhindra3000) [reply lib/pages/stock_exchange/components/stock_detail.dart:127]: Everything else is good to go 👍

**Review by Sudhindra3000 — CHANGES_REQUESTED** [review]

File: lib/pages/company_page/components/trading_bottom_sheet.dart [hunk 2]
```diff
@@ -20,6 +21,8 @@ import 'package:intl/intl.dart';
 
 final oCcy = NumberFormat('#,##0.00', 'en_US');
 
+// TODO: stopLoss order ui rebuilding due to sheet keyboard issue
+// TODO: fix placing orders without giving price and handle all the edge cases
```
Comment (siva2204) [inline lib/pages/company_page/components/trading_bottom_sheet.dart:25]: i dont understand this ?

Comment (dhruv1294) [reply lib/pages/company_page/components/trading_bottom_sheet.dart:25]: try to give a stopLossOrder in mobile

Comment (dhruv1294) [reply lib/pages/company_page/components/trading_bottom_sheet.dart:25]: when keyboard opens, sheet rebuilds again and changes to limit order

Comment (Sudhindra3000) [reply lib/pages/company_page/components/trading_bottom_sheet.dart:25]: > i dont understand this ?

problem

**Pull request merged**